### DISCUSSION TITLE
website security

### DISCUSSION THREAD

**Discussion by OscarNguyen** [community/community#19001] [Code Security]
how to make a website as secure as possible?

**Comment by Andreaa-Dev** [comment]
use https instead of http ☺️

**Reply by OscarNguyen** [reply]
also a nice way, bruh :))

**Comment by Zerotask** [comment]
There is no universal answer to this. TLS is one option you should always consider to avoid MITM attacks. There are also web application firewalls. Most of the danger comes from the app or the developer itself. There are a lot of best practices like never trust the client (always validate user input), open only ports you need from outside etc.

**Reply by OscarNguyen** [reply]
thanks, what a full answer it is ^^

**Comment by webgtx** [comment]
> how to make a website as secure as possible?

Use automated scripts to find vulnerabilities.
Also you can pentest website by your own hands with **metasploit**
